mvrx/build.gradle

```diff
@@ -52,17 +52,18 @@ jacoco {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
```

Comment:
@elihart @gpeal `kotlin-stdlib` is a dependency of `kotlin-stdlib-jdk8` and the previous merge bumped the JVM to Java 8. Do you think is required keep both versions?

I ask you that because I have removed it during the refactor and made `implementation libraries.kotlin` point **only** to `kotlin-stdlib-jdk8` :thinking:.

Reply:
you're right, we should only need `kotlin-stdlib-jdk8` as it includes `kotlin-stdlib`